packages/visual-editor/src/api/nearbyLocations.tsx

```diff
@@ -0,0 +1,57 @@
+import { getCurrentDateYYYYMMDD } from "../utils/dateUtils.ts";
+
+export const fetchNearbyLocations = async ({
+  businessId,
+  apiKey,
+  latitude,
+  longitude,
+  radiusMi,
+  entityType,
+}: {
+  businessId: number;
+  apiKey: string;
+  longitude: number;
+  latitude: number;
+  radiusMi: number;
+  entityType?: string;
+}): Promise<any> => {
+  const base = `https://streams-dev.yext.com/v2/accounts/${businessId}/content/visualEditorLocations`;
+  const queryParams: any = {
+    api_key: apiKey,
+    v: getCurrentDateYYYYMMDD(),
```

Comment:
You need to set an actual date. If you always get the latest date and the api changes then it would be breaking. Date pins to a specific version of the api.